packages/arkor-local/python/mlx/dataset_prep.py

```
@@ -0,0 +1,254 @@
+        # asked for exactly one split).
+        for candidate in ("validation", "valid"):
+            try:
+                valid = load(candidate)
+                log(f"[arkor] using the dataset's {candidate!r} split for eval")
+                break
+            except Exception:
+                continue
+    return train, valid
+
+
+def _load_blob(source: dict, log):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_load_blob`

(ANN202)